src/config/jsonconfig.rs

```diff
@@ -13,6 +13,7 @@ pub struct JsonConfig {
     pub description: Option<String>,
 
     pub theme_path: Option<PathBuf>,
+    pub curly_quotes: Option<bool>,
```

Comment:
The only reason the JSON config is still there is to give people the time to migrate to the TOML one. So, I'm not sure if I would add the new configuration options to it.